### PR TITLE
Renovated component: options set to undefined should fall it back to default options considering defaultOptionRules

### PR DESCRIPTION
**Pull request by ksercs**
…

### CODE REVIEW

File: js/renovation/component_wrapper/component.ts [hunk 1]
```diff
@@ -158,23 +151,53 @@ export default class ComponentWrapper extends DOMComponent {
     return this._elementAttr;
   }
 
+  _setDefaultOptionValue(options, defaultValueGetter) {
```
Comment (pavelgruba) [inline js/renovation/component_wrapper/component.ts]: This method does not use `this`, so why is it an instance method now?

Comment (ksercs) [reply js/renovation/component_wrapper/component.ts]: just forgot to move it out after some changes, thx

**Comment by ksercs** [comment]
waits for generator improving

**Comment by ksercs** [comment]
will be merged only in 21.2

**Pull request closed without merging**